package.json

```diff
@@ -10,7 +10,8 @@
   "scripts": {
```

Comment:
let's bump the version above to `1.2.0` and we can tag it once we have this merged, then we can use the tag in our package.json